libsolidity/formal/CHC.cpp

```diff
@@ -642,10 +642,18 @@ void CHC::externalFunctionCall(FunctionCall const& _funCall)
 	}
 
 	auto postCallState = vector<smtutil::Expression>{state().state()} + currentStateVariables();
-	auto nondet = (*m_nondetInterfaces.at(m_currentContract))(preCallState + postCallState);
+	auto error = errorFlag().increaseIndex();
+	auto nondet = (*m_nondetInterfaces.at(m_currentContract))(preCallState + postCallState + vector<smtutil::Expression>{error, state().thisAddress()});
 	// TODO this could instead add the summary of the called function, where that summary
 	// basically has the nondet interface of this summary as a constraint.
 	m_context.addAssertion(nondet);
+	solAssert(m_errorDest, "");
+	connectBlocks(m_currentBlock, predicate(*m_errorDest), errorFlag().currentValue() > 0);
+	solAssert(m_currentContract && m_currentFunction, "");
+	for (auto const* base: m_currentContract->annotation().linearizedBaseContracts)
+		for (auto const* defined_function: base->definedFunctions())
```

Comment:
```suggestion
		for (auto const* function: base->definedFunctions())
```

Reply:
This would cause a name clash, there already is a local variable `function`. Is that OK, or should that local variable be renamed?

Reply:
Please just rename it to `definedFunction` then (style)

Reply:
Ah, I was completely blind to the style difference...
Fixed.